packages/back-end/src/services/auth/OpenIdAuthConnection.ts

```diff
@@ -37,6 +37,8 @@ if (USE_PROXY) {
   custom.setHttpOptionsDefaults(getHttpOptions());
 }
 
+const passthroughQueryParams = ["hypgen", "hypothesis"];
```

Comment:
`hypgen=1` triggers the new layout
`hypothesis=foo+bar` renders the hypothesis text onto the page

Reply:
Is it possible to consolidate into `hypothesis`? (Check if it's non-empty for new layout)

Reply:
it is, but thought it might be good to keep things separate

Reply:
(also bc we'll eventually import multiple hypotheses/variants at once for bandits, in which case we likely wouldn't render hypothesis text)

Reply:
Seems premature but I'll allow it 🤷🏾